modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -98,6 +99,7 @@
     protected boolean hateoas = false;
     protected boolean returnSuccessCode = false;
     protected boolean unhandledException = false;
+    protected boolean useSpringJpa = false;
```

Comment:
@MarkusBansky And here, we could rename this property from `useSpringJpa` to `useJpa`?